src/main/java/org/jabref/logic/util/io/FileUtil.java

```diff
@@ -284,7 +284,7 @@ public static String createDirNameFromPattern(BibDatabase database, BibEntry ent
         String targetName = BracketedPattern.expandBrackets(directoryNamePattern, ';', entry, database);
 
         if (targetName.isEmpty()) {
-            targetName = entry.getCiteKeyOptional().orElse("default");
+            return targetName;
```

Comment:
Wow, what a mistake in the code. I checked the method names and comment. The change seems to be valid.